crates/router/src/core/payments.rs

```diff
@@ -5981,6 +5981,76 @@ where
     Ok(final_list)
 }
 
+pub struct SessionTokenRoutingResult {
+    pub final_result: Vec<api::SessionConnectorData>,
+    pub routing_result:
+        FxHashMap<common_enums::PaymentMethodType, Vec<api::routing::SessionRoutingChoice>>,
+}
+#[cfg(feature = "v2")]
+pub async fn perform_session_token_routing<F, D>(
```

Comment:
If this is routing relate function, move from payments to routing file.

Reply:
No there is another function from routing module `let result = self_routing::perform_session_flow_routing(sfr, &enums::TransactionType::Payment)` used inside this function. I think this one belongs in payments module only.